.github/workflows/ci.yml

```diff
@@ -38,3 +38,25 @@ jobs:
             echo "README.md is out of date. Please run helm-docs and commit the changes."
             exit 1
           fi
+
+      - name: Check that chart version is bumped if chart appVersion is bumped
+        run: |
+          CHART_DIR="charts/deepgram-self-hosted"
+          if [ ! -f "$CHART_DIR/Chart.yaml" ]; then
+            echo "Error: Chart.yaml not found in $CHART_DIR"
+            exit 1
+          fi
+
+          OLD_VERSION=$(git show $GITHUB_BASE_REF:$CHART_DIR/Chart.yaml | grep '^version:' | awk '{print $2}')
+          NEW_VERSION=$(grep '^version:' $CHART_DIR/Chart.yaml | awk '{print $2}')
+          OLD_APP_VERSION=$(git show $GITHUB_BASE_REF:$CHART_DIR/Chart.yaml | grep '^appVersion:' | awk '{print $2}')
+          NEW_APP_VERSION=$(grep '^appVersion:' $CHART_DIR/Chart.yaml | awk '{print $2}')
+
+          if [[ "$OLD_VERSION" = "$NEW_VERSION" ]] && [[ "$OLD_APP_VERSION" != "$NEW_APP_VERSION" ]]; then
+            echo "Error: In $CHART_DIR/Chart.yaml, appVersion has been changed but version has not been bumped"
+            echo "Old appVersion: $OLD_APP_VERSION"
+            echo "New appVersion: $NEW_APP_VERSION"
+            echo "Chart version: $NEW_VERSION"
+            echo "Chart version should be bumped anytime the application version is bumped."
```

Comment:
Is this bump done manually, or through helm-docs?

Reply:
Helm-docs just updates the README based on the contents of the Chart.yaml. Ultimately, the source of truth for the version is going to be the Chart.yaml, which needs to be manually edited. 

Update scenarios for these versions could include:
- No version bump. We are making changes that we don't want to release yet, but want to merge. 
- Bump only the chart version. This could include an update to the templates that doesn't bump the container tags (which map to the Deepgram self hosted release and is what we use for the appVersion)
- Bump both the chart and app version for an update that includes changing the container tags. 

Any other combination of version bump, like updating the app version but not chart version, shouldn't be allowed.